### PR TITLE
Delete ui

### PR DESCRIPTION
**Pull request by vinhtruong94**

### CODE REVIEW

**Comment by vinhtruong94** [comment]
@jessicaquynh check this

File: public/app/components/user/editUser/editUser.dir.js [hunk 1]
```diff
@@ -13,20 +13,30 @@
 			 'userRoleResource',
 			 'userRoleFactory',
 			 'Authorizer',
+			 'ToastMessage',
 			 'utils',
 		editUser]);
 
 	/** @ngInject */
+<<<<<<< HEAD
```
Comment (Kiniamaro) [inline public/app/components/user/editUser/editUser.dir.js]: wop wop wop

File: public/app/components/user/editUser/editUser.dir.js [hunk 2]
```diff
@@ -13,20 +13,30 @@
...
+	function editUser(communityResource, editUserFormService, userResource, userRoleResource, userRoleFactory, Authorizer,ToastMessage, utils) {
+=======
 	function editUser(communityResource, editUserFormService, userResource, userPreferenceResource, userPreferenceFactory, userRoleResource, userRoleFactory, Authorizer, utils) {
+>>>>>>> ffca5938c61432abad24d15e555be2942b0d93f4
```
Comment (Kiniamaro) [inline public/app/components/user/editUser/editUser.dir.js]: unfixed merge conflict

Comment (vinhtruong94) [inline public/app/components/user/editUser/editUser.dir.js]: ohhhh darn it !

Comment (vinhtruong94) [inline public/app/components/user/editUser/editUser.dir.js]: Thanks @Kiniamaro  !

**Pull request merged**